server/app/filters/UnsupportedBrowserFilter.java

```diff
@@ -0,0 +1,39 @@
+package filters;
+
+import java.util.Optional;
+import play.libs.streams.Accumulator;
+import play.mvc.Call;
+import play.mvc.EssentialAction;
+import play.mvc.EssentialFilter;
+import play.mvc.Http;
+import play.mvc.Results;
+
+/**
+ * Filter that redirects requests from unsupported browsers (e.g. IE11) to a page that tells user to
+ * use newer browser.
+ */
+public class UnsupportedBrowserFilter extends EssentialFilter {
+  @Override
+  public EssentialAction apply(EssentialAction next) {
+    return EssentialAction.of(
+        request -> {
+          if (UnsupportedBrowserFilter.isUnsupportedBrowser(request)) {
+            Call unsupportedBrowser =
+                controllers.routes.SupportController.handleUnsupportedBrowser();
+            // Don't redirect if request is for a static asset or for the unsupported browser page
```

Comment:
Nit and not blocking: Consider moving this clause above the isUnsupportedBrowser check and early out.

Reply:
Do you think that will be faster? Because it will remove all static requests faster and we have a lot of those?

Reply:
My original rationale was more of a readability / style preference to avoid an additional level of indentation.

Having an early out feels a bit more in fitting with the idea that "none of the logic in this filter is applied for static assets".

Totally up to you on that though, my preference is weak.